.github/workflows/stale.yml

```diff
@@ -3,17 +3,15 @@ on:
   schedule:
     - cron: "16 12 * * *"
   workflow_dispatch:
-
 jobs:
   stale:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/stale@b12dccced80582bf8c52244f75838892ec5c1e82
       with:
         repo-token: ${{ secrets.GITHUB_TOKEN }}
-        stale-pr-message: 'This pull request is inactive and will be closed'
-        skip-stale-pr-message: true
-        close-pr-message: 'This pull request has been inactive for more than 30 days and has been automatically closed. Feel free to register your package or version again once you fix AutoMerge issues'
+        stale-pr-message: 'This pull request has been inactive for 30 days and will be automatically closed 7 days from now. Post a comment if this pull request should not be closed.'
```

Comment:
Rather than generically prompting the user to comment, I'd encourage them to take some action, like fixing automerge issues or explaining why they would like to skip a version or use a non-automatically-accepted name.  99% of the stale PRs are just waiting an active action from the author.  Action that could have been taken also 30 days before, so I'm not sure waiting 7 more days helps that much

Reply:
> Action that could have been taken also 30 days before, so I'm not sure waiting 7 more days helps that much

The AutoMerge guidelines are not actually required for merging (see discussion in https://github.com/JuliaRegistries/General/issues/10163).

So the PR author could just be waiting for someone to manually merge the PR.

Reply:
> Rather than generically prompting the user to comment, I'd encourage them to take some action, like fixing automerge issues or explaining why they would like to skip a version or use a non-automatically-accepted name.

This is a good idea.

Reply:
> The AutoMerge guidelines are not actually required for merging (see discussion in #10163).

Sure, but manual merge is at discretion of maintainers.  Most stale PRs miss proper compat bounds, and I discretionally don't manually merge them